### DISCUSSION TITLE
Transform requires an owner - error message

### DISCUSSION THREAD

**Discussion by bilbonl** [community/community#33636] [Other Feature Feedback, Questions, & Ideas]
Hi,

I'm trying to change a user repo to an organization repo. I fill in all the details and choose a user that will own the organization, but I keep getting an error "Transform requires an owner."

Any idea would be appreiacted.

Thanks!

**Comment by 123456-111** [comment]
I think voron v2.4 kit parts have few worth thing.
It make some critical trobles, User dont know this problem.
but,there must resolve the problem.

**Reply by bilbonl** [reply]
I'm not sure I understand...

**Comment by LanceMcCarthy** [comment]
Selecting the name doesn't actually add them yet. You need to click the `Add Owner` button after selecting them. I know not intuitive, but I can confirm that it works.

> [!NOTE]
> July 2024 Update: As @ruijiang mentions, the `Add Owner` button now shows the text `Choose`.

**Reply by AppExert** [reply]
True, not intuitive at all. The above suggestion works. Thanks @LanceMcCarthy

**Reply by ruijiang** [reply]
Wow. Thanks so much! Today the button is called "Choose". I had no idea I need to click that.

**Comment by banagale** [comment]
Piping up that I just filled out this form three times with a colleague before finding this page.  

Github needs to:
1. Save the contents of the form on submit (Please)
2. Give a proper error message, saying the user needs to click Choose after selecting a user name.  

"Transform requires an owner" error message does not map to the UX requirements and the reset of the form causes the visual context to be lost completely.

This would probably not be a big deal, except getting to a long lost user's account that needs to become an organization can be an ordeal enough.  

Please clean up this UX.

**Comment by maxxlife** [comment]
Thanks, it still helps, and needs to improve UI. You actually must click "Choose"

**Comment by icterra-official** [comment]
Hi, I have different issue, I have Github account and I try to add as admin but I get error, "this user doesn't have Github account".
Have any idea?
